core/src/test/java/org/infinispan/test/AbstractInfinispanTest.java

```diff
@@ -91,16 +91,16 @@ protected void eventuallyEquals(int expected, IntSupplier supplier) {
             () -> expected == supplier.getAsInt());
    }
 
-   protected void eventuallyEquals(String message, long expected, LongSupplier supplier) {
+   public static void eventuallyEquals(String message, long expected, LongSupplier supplier) {
```

Comment:
I don't like mixing static and non-static methods in the same class, especially when they have so similar names, so I'd rather move these to a separate class, e.g. `Eventually`. 

Or, if you don't want to affect the core tests, you could just replace your `TestingUtil` with an `AbstractStatisticsTest` extending `AbstractInfinispanTest`.